app/models/developer_external_profile.rb

```diff
@@ -0,0 +1,15 @@
+class DeveloperExternalProfile < ApplicationRecord
+  enum :sites, %i[linkedin github]
```

Comment:
Let's remove `github` since we aren't using it yet. But thanks for taking into account that we will in the future!